DVLSForLinux/install-dvls.sh

```diff
@@ -12,30 +11,24 @@ show_usage() {
   echo "  --database-host <DatabaseHost>           Specify the database host"
   echo "  --database-username <DatabaseUsername>   Specify the database username"
   echo "  --database-password <DatabasePassword>   Specify the database password"
+  echo "  --database-name <DatabaseName>           Specify the database name"
+  echo "  --zip-file <ZipFile>                     Specify a zip file for the DVLS installation file"
   echo
   echo "Flags:"
   echo "  -h, --help                               Show this help message and exit"
   echo "  -y, --no-confirm                         Do not confirm the action before proceeding"
-  echo
-  echo "  --database-encrypted-connection          Enable encrypted connection to the database"
-  echo "  --no-database-encrypted-connection       Disable encrypted connection to the database"
-  echo
+  echo "  --database-encrypted-connection          Enable or disable encrypted connection to the database"
```

Comment:
@alistek In bash, it’s only used to enable, it’s not possible to disable
```suggestion
  echo "  --database-encrypted-connection          Enable encrypted connection to the database"
```